src/components/UI/FormItems/TextField.js

```diff
@@ -3,24 +3,36 @@ import PropTypes from 'prop-types';
 import MUITextField from '@material-ui/core/TextField';
 import FormControl from '@material-ui/core/FormControl';
 import FormHelperText from '@material-ui/core/FormHelperText';
+import InputAdornment from '@material-ui/core/InputAdornment';
 
 const propTypes = {
   className: PropTypes.string,
   error: PropTypes.oneOfType([
     PropTypes.bool,
     PropTypes.string,
     PropTypes.number
-  ])
+  ]),
+  endAdornment: PropTypes.node
 };
 
 const defaultProps = {
   className: '',
-  error: false
+  error: false,
+  endAdornment: null
 };
 
-const TextField = ({ className, error, ...props }) => (
+const TextField = ({ className, error, endAdornment, ...props }) => (
   <FormControl className={className} error={!!error}>
-    <MUITextField error={!!error} {...props} />
+    <MUITextField
+      error={!!error}
+      {...props}
+      InputProps={{
```

Comment:
If you make the changes that i suggested use  `InputProps={InputProps}` instead.